saleor/payment/utils.py

```diff
@@ -80,6 +86,8 @@ def create_payment_information(
         reuse_source=store_source,
         data=additional_data or {},
         graphql_customer_id=graphql_customer_id,
+        store=StoreEnum[payment.store.upper()],
```

Comment:
MyPy doesn't like `store=payment.store`, but other than that it works. Should I just suppress the type checking for this statement?

Reply:
MyPy doesn't like it because `payment.store` is an enum value, and `store` expects to receive str. So `paymentData.store` should have a different type, or you can also convert `payment.store` to `payment.store.value`

Reply:
Type of `payment.store` is a `str` and `PaymentData.store` expects `StoreEnum`. I use `StoreEnum[payment.store.upper()]` just to convert a type